apps/backend/src/modules/vendors/libs/mapper/edx-course-fields.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { EdxCourseField } from "~/modules/vendors/libs/enums/enums.js";
+import {
+	type CourseFieldsMapping,
+	type EdxCourseResponseDto,
+} from "~/modules/vendors/libs/types/types.js";
+
+const EdxCourseFieldsMap: CourseFieldsMapping = {
```

Comment:
Vlad taught me, and now I can pass on this knowledge to you!
https://dev.to/what1s1ove/javascript-naming-conventions-are-important-45n
```suggestion
const edxCourseToCourse: CourseFieldsMapping = {
```